third-party/realdds/src/dds-device-server.cpp

```diff
@@ -91,9 +94,9 @@ static void on_discovery_stream_header( std::shared_ptr< dds_stream_server > con
         { "default-profile-index", stream->default_profile_index() },
         { "metadata-enabled", stream->metadata_enabled() },
     } );
-    LOG_DEBUG( "-----> JSON = " << stream_header_message.json_data().dump() );
-    LOG_DEBUG( "-----> JSON size = " << stream_header_message.json_data().dump().length() );
-    LOG_DEBUG( "-----> CBOR size = " << json::to_cbor( stream_header_message.json_data() ).size() );
+    auto json_string = slice( stream_header_message.custom_data< char const >(), stream_header_message._data.size() );
+    LOG_DEBUG( "-----> JSON = " << shorten_json_string( json_string, 300 ) << " size " << json_string.length() );
```

Comment:
Why 300? I think that if we shorten it than about a line width, 120 chars, should be better

Reply:
We have lots of info, so this depends on the message, I found 300 to be not too long and long enough that you could see the top-level json elements and their values.

Reply:
OK